examples/ML_classifier_copies/ML_copies_integration.ipynb

```diff
@@ -14,7 +14,7 @@
    "outputs": [],
    "source": [
     "import sys\n",
-    "sys.path.append(\"../\")\n",
+    "sys.path.append(\"../../\")\n",
```

Comment:
(Here and in the other notebooks) Is this necessary? The presc package should be already be pip-installed?